dcos/package.py

```diff
@@ -2045,6 +2059,23 @@ def uninstall_app(self, app_name, remove_all, app_id):
         response = http.post(url, json=params)
         return response.status_code == 200
 
+    def search_sources(self, query):
+        """package search
+
+        :param query: query to search
+        :type query: str
+        :returns: list of package indicies of matching packages
+        :rtype: [packages]
+        """
+        url = urllib.parse.urljoin(self.cosmos_url, 'v1/package/search')
+        response = http.post(url, json={"query": query})
+
+        if response.status_code != 200:
+            raise DCOSException(response.json().get("message"))
+
+        packages = response.json()
+        return [{"packages": packages, "source": "cosmos"}]
```

Comment:
no, it's really just a placeholder until we add support for sources. ill add an issue for it